source/functions/mongodb/api.txt

```diff
@@ -56,10 +89,76 @@ you to access collections in the specified database.
 
 See :method:`database.collection()`
 
+.. _mongodb-admin-getdbnames:
+
+admin.getDBNames()
+------------------
```

Comment:
this the only method for admin object right now?

Reply:
Yup currently the only one but I worded it to be generic should we add more in the future.